addons/edi_stock/data/edi_move_cancellation_data.xml

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0"?>
+<odoo>
+    <data>
+        <!-- Processed Purchase Orders sequence -->
+        <record id="sequence_edi_move_cancellation" model="ir.sequence">
+            <field name="name">Move Cancellation</field>
+            <field name="prefix">MCAN</field>
+            <field name="padding" eval="5" />
+        </record>
+        <!-- Processed Purchase Order record type -->
+        <record id="edi_record_type_move_cancellation" model="edi.record.type">
+            <field name="name">Move Cancellation</field>
+            <field name="sequence">67</field>
+            <field name="model_id" ref="model_edi_record_stock_cancellation" />
+        </record>
+        <!-- Processed Purchase Orders document type -->
+        <record id="edi_document_type_move_cancellation" model="edi.document.type">
+            <field name="name">Move Cancellation</field>
+            <field name="model_id" ref="model_edi_document_stock_cancellation" />
+            <field name="sequence">23</field>
+            <field name="sequence_id" ref="sequence_edi_move_cancellation" />
+            <field name="rec_type_ids" eval="[(6, False, [
+          ref('edi_record_type_move_cancellation'),
+        ])]" />
+            <!-- pick_type_ids will be determined based on which ones have
+            u_enable_pick_cancellation_email_notifs set -->
+        </record>
+    </data>
+
+    <data noupdate="1">
+        <!-- Intended as a baseline template for Move Cancellation EDI, in a noupdate
+        block so that recipients can be configured without being overridden on upgrade -->
+        <record id="move_cancellation_template" model="mail.template">
+            <field name="name">Move Cancellation</field>
+            <field name="model_id" ref="model_edi_record_stock_cancellation" />
+            <field name="auto_delete" eval="False" />
+            <field name="email_from">${user.name} &lt;${user.email | safe}&gt;</field>
+            <field name="subject">Move Cancellation for ${object.sale_id.client_order_ref | safe}</field>
+            <field name="body_html">
+                <![CDATA[
+        <p>Products have been cancelled from a move related to <span style="font-weight: bold;">${object.sale_id.client_order_ref}</span> for the following customer:</p>
+        <p>${object.customer_id.name}<br/>
+            % if object.customer_id.street:
```

Comment:
i couldn't find any nice examples of address formatting in jinja, so opted to do this. if anyone knows better ways, let me know!

Reply:
I don't think there's a nice way to do many things in jinja tbh. :smile: 

Maybe long term we could look into adding a computed field, something like `formatted_address` which would take care of this for us and just let us use one line.

Reply:
I will look into `contact_address` on res.partner.

Reply:
so `contact_address` works mostly, but it uses `\n` rather than `<br/>`. I am pretty (90%) sure that  `\n` isn't a valid newline for emails
I tried ```${object.customer_id.contact_address.replace("\n", "<br/>")}``` but this doesn't parse the line breaks
![image](https://user-images.githubusercontent.com/10167528/128418589-a6049b5b-b293-427a-aec1-c39db7a13f74.png)

When hardcoding them in the if statements they get parsed as line breaks properly, it's probably escaping them to a string or something and preventing them from actually producing dom output..

TIL: `| safe` prevents odoo escaping db values.. So with it users can screw up the DOM by putting jinja code in their address (shrug)
`<p>${object.customer_id.contact_address.replace("\n", "<br/>") | safe}</p>` gives me the result i want, at the expense of jinja injection :-1:

Reply:
That's definitely annoying! But thanks for looking into that. :)

Maybe we can just raise an internal issue, it's not a blocker and certainly shouldn't hold up this story. :)